src/test/java/reposense/commits/CommitInfoAnalyzerTest.java

```diff
@@ -201,7 +231,7 @@ public void analyzeCommits_emptyCommits_success() throws ParseException {
 
         expectedCommitResults.add(new CommitResult(author, "016ab87c4afe89a98225b96c98ff28dd4774410f",
                 parseGitStrictIsoDate("2020-01-27T22:20:51+08:00"), "empty commit",
-                "", null, 0, 0));
+                "", null, 0, 0, new HashMap<>()));
```

Comment:
If the expected fileType contribution here is empty, then is it possible to use `Collections.emptyMap()` here?

```suggestion
                "", null, 0, 0, Collections.emptyMap()));
```